src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -437,15 +437,15 @@ protected override void OnInitializeConfig(ScriptHostConfiguration config)
             hostConfig.AddService<IWebHookProvider>(this._bindingWebHookProvider);
             hostConfig.AddService<IWebJobsExceptionHandler>(_exceptionHandler);
 
-            // disable standard Dashboard logging (enabling Table logging below)
-            hostConfig.DashboardConnectionString = null;
```

Comment:
I had reordered this, and the problem was that it then resulted in loggingConnectionString being null when we created the FunctionInstanceLogger. We need that dashboard connection string still for the table logging. Moving the explicit set to null back where it was, at the end of the method.